.env.example

```diff
@@ -3,11 +3,12 @@ AWS_SECRET_ACCESS_KEY=aBcDeFgHiJkLmNoPqRsTuVwXyZAbCdEfGhIjKlMn
 AWS_DEFAULT_REGION=us-east-1
 AWS_REGION=us-east-1
 
-CF_API_EMAIL=cio@domain1.org
 CF_API_KEY=1234abcd
+CF_API_KEY_ETH=1234abcd
```

Comment:
Do you want to name this something more generic, like `CF_API_KEY_2`?

Reply:
Yeah, that's probably better. I was following precedent, but I like your suggestion.